### DISCUSSION TITLE
404 error solution

### DISCUSSION THREAD

**Discussion by mayooear** [mayooear/ai-pdf-chatbot-langchain#11] [Announcements]
In this issue @ankri noted the following based on trial and error:

"For future readers: Yes. You are running into problems with text-davinci-003 because the OpenAI wrapper is hitting the https://api.openai.com/v1/chat/completions endpoint.

These are the supported models for this endpoint:
image
gpt-4, gpt-4-0314, gpt-4-32k, gpt-4-32k-0314, gpt-3.5-turbo, gpt-3.5-turbo-0301

[Source](https://platform.openai.com/docs/models/gpt-3)

**Comment by biggleskc** [comment]
Hi, I keep getting the 404 error. Is there a suggested fix?

**Comment by SilverNotGold** [comment]
Getting the 404 error, anyone got a solution ? Not understanding the response above, I'm using gpt-4
